pkg/plugin/nvidia/nvidia.go

```diff
@@ -119,13 +132,23 @@ func checkHealth(stop <-chan struct{}, devices []*Device, unhealthy chan<- *Devi
 
 		e, err := nvml.WaitForEvent(eventSet, 5000)
 		if err != nil && e.Etype != nvml.XidCriticalError {
+			if firstTime {
```

Comment:
Would you describe what is your case?  Why we have to patch unhealthy for the first time?

Reply:
It use to reset node annotations if all device healthy, when device plugin restart.

Reply:
@william-wang